packages/react/src/components/form/form.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { Form } from ".";
+import { ConfigurationProvider } from "../../context/config-context";
+
+const meta: Meta<typeof Form> = {
+  component: Form,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Form>;
+
+export const Primary: Story = {
+  render: () => (
+    <ConfigurationProvider>
```

Comment:
I don't think that the configuration provider is mandatory, so this feels extraneous/confusing for a new user.

Maybe we should remove it?

Reply:
Agree with Jake here, initial stories should have bare minimum, ideally only required props. We can add more stories to showcase advanced config options later

Reply:
Yes that makes more, thank you!

Reply:
@jake-slashid I just checked the [docs](https://developer.slashid.dev/docs/react-sdk/reference/components/react-sdk-reference-form) and it say that **_a `<ConfigurationProvider>` needs to be rendered as a parent component_**, doesn't that mean that it's mandatory for the `Default` story?

Reply:
I'm building a demo at the moment and thought I did it without the configuration provider, looking at the source code its pretty clear the configuration context is required.

We should probably change this but for now, please ignore me and include it in the example!

Reply:
@jake-slashid I tried using the same example for `<SlashIDLoaded>` [here](https://developer.slashid.dev/docs/react-sdk/reference/components/react-sdk-reference-slashidloaded) but it says that it doesn't accept text as child elements:
```
    <SlashIDLoaded>
      SlashID is ready!
    </SlashIDLoaded>
```

Should we update the docs or change the children type to `React.ReactNode`?

Reply:
There's a PR for it here: https://github.com/slashid/javascript/pull/106